docs/source/point_data/point_methods.rst

```diff
@@ -41,12 +41,12 @@ attributes are available in :ref:`point_obs_metadata`. ::
     data_source = 'usgs_nwis'
     variable = 'streamflow'
     temporal_resolution = 'daily'
-    aggregation = 'average'
+    aggregation = 'mean'
```

Comment:
why did this change from average to mean?

Reply:
Good question! In December, this convention from calling it "average" to calling it "mean" changed when we were reconciling the point parameter values with how things are named for the gridded parameters. It looks like this particular example slipped through when we updated things with that new convention.